package.json

```diff
@@ -27,18 +27,18 @@
     "@airtap/browserify-istanbul": "^4.0.0",
     "airtap-default": "^1.0.0",
     "airtap-multi": "^1.0.0",
-    "browserify": "^16.5.2",
+    "browserify": "^17.0.0",
```

Comment:
Because of this upgrade, airtap users will have to take an additional step to test on IE11. Can you please update [this readme section](https://github.com/airtap/airtap#ie--11-support)? It should say:

> To support IE < 11, older versions of the [`buffer`](https://github.com/feross/buffer) and [`stream-browserify`](https://github.com/browserify/stream-browserify) polyfills are required. Use the following configuration and run `npm install buffer@4 stream-browserify@2`:
>
>```yaml
> # Use buffer@4 and stream-browserify@2 to support IE < 11
> browserify:
>  - require: 'buffer/'
>    expose: 'buffer'
>  - require: 'stream-browserify'
>    expose: 'stream'
>```